gqlalchemy/how-to-guides/loaders/import-table-data-to-graph-database.md

```diff
@@ -0,0 +1,121 @@
+---
+id: table-to-graph-importer
+title: How to import table data to graph database
+sidebar_label: Import table to graph
+---
+
+This guide will show you how to use `loaders.py` to translate table data from a file to graph data and import it to Memgraph. Currently, we support reading of CSV, Parquet, ORC, IPC, Feather and Arrow file formats via Pyarrow. Data loading is implemented from `LocalFileSystem`, `AzureBlobFileSystem` and `AmazonS3FileSystem`.
+
+> Make sure you have a running Memgraph instance. If you're not sure how to run
+> Memgraph, check out the Memgraph [Quick start](/memgraph/#quick-start).
+
+## Loading a CSV file from `LocalFileSystem`
```

Comment:
Since this how-to has two sections, I would either divide it into two how-to's or add a simple overview linking to those two sections before this heading. Smth like here: https://memgraph.com/docs/memgraph/import-data/csv-import-tool#examples